### DISCUSSION TITLE
Lesson6: Command 'register-python-argcomplete' not found

### DISCUSSION THREAD

**Discussion by fushengcheng** [Cyfrin/foundry-full-course-cu#614] [Q&A]
This comes up after I run the foundry install command in WSL.
```
Command 'register-python-argcomplete' not found, did you mean:

  command 'register-python-argcomplete3' from deb python3-argcomplete (1.8.1-1.3ubuntu1)

Try: sudo apt install <deb name>
```
The command that wasn't found came from this line in the .bashrc file.
```
eval "$(register-python-argcomplete ./console.py)"
```
This prompt doesn't seem to affect the foundry installation, but it appears every time when I start the terminal. 
I'm wondering if I change "register-python-argcomplete" to "register-python-argcomplete3" if that affects the use of foundry?
How can I solve this problem?

**Comment by theirrationalone** [comment]
Python isn't installed in your PC.
